api_app/api/routes/api.py

```diff
@@ -8,10 +8,15 @@
 from api.helpers import get_repository
 from db.repositories.workspaces import WorkspaceRepository
 from api.routes import health, ping, workspaces, workspace_templates, workspace_service_templates, user_resource_templates, \
-    shared_services, shared_service_templates, migrations, costs, airlock, operations, metadata, requests
+    shared_services, shared_service_templates, migrations, costs, airlock, operations, metadata, requests, workspace_users
 from core import config
 from resources import strings
 
+
+def _is_user_management_enabled():
```

Comment:
Does this really need it's own method, can we just call directly, liek we do with others, such as `config.API_DESCRIPTION` elsewhere in the file?

Reply:
This was only there to make testing easier, happy to revert

Reply:
removed